pkg/sloop/webfiles/sloop_ui.js

```diff
@@ -291,9 +314,9 @@ function getHeatmapContent(d) {
     let allReasons = d.text.split(" ").reduce((r, l, i, a) => {
         let splitText = l.split(":")
         return `<tr>
-                 <td> <b style=color:${eventKindColorGenFunc(splitText[0])}>${splitText[0]}</b> </td>
+                 <td> <b style=${splitText[0]}></b> </td>
                  <td> <b> ${splitText[2]} </b> </td> 
-                 <td> <b style="color:${severityColorGenFunc(severity.get(splitText[1]))}">${splitText[1]}</b> </td>
+                 <td> <b style="color:${severity.get(splitText[1])}">${splitText[1]}</b> </td>
```

Comment:
this seems off
I'm assuming the `splitText[1]` is the error level (Normal, Warning or Error). The `.get()` call translates the text to a number. How does this get translated to the color?

Reply:
it might be helpful to use a local variable to provide a name
`let errorLevel = splitText[1]`